### ISSUE TITLE
getting a warning while building the imported embARC example project in GNU IDE 2020.03-RC1

### ISSUE THREAD

**Issue by joejohnpaulines** [foss-for-synopsys-dwc-arc-processors/toolchain#265]
We are getting a below arc plugin warning when we build an imported the embARC 2019.06 example project on GNU 2020.03 ide.  Please check the details here.

![image](https://user-images.githubusercontent.com/52485302/80681886-2c12ef80-8adf-11ea-835c-10fdfb69a016.png)

**Tools Version** 

embARC 2019.06 :  https://github.com/foss-for-synopsys-dwc-arc-processors/embarc_osp/archive/2019.06.tar.gz
GNU IDE - 2020.03-RC1

Steps that I followed:

1. Extracted 2019.06.tar.gz and use "C:\Users\joejohn\Downloads\embarc_osp-2019.06_v3\example\baremetal\arc_feature\timer_interrupt" example.

2. Import the timer_interrupt example to the IDE project [  New - > Makefile project with existing code -> [ Project name = example_1 , Exisitng code location : C:\Users\joejohn\Downloads\embarc_osp-2019.06_v3\example\baremetal\arc_feature\timer_interrupt, Languages = select c, c++, Toolchain for indexer setting = none ] -> click finish

3. Add the following to the makefile of the imported project.

        BOARD=nSIM
        BD_VER=10 
        TOOLCHAIN=gnu
4. build the project.

Observation

 - In Linux [ centos 7.3 ], we are getting the warning message, but we are able to build the project successfully.
- in windows 10, we are getting that warning message, We are not able to build the project and we are getting below error message.

>  make all 
> ../../../../options/scripts.mk:9: *** missing separator.  Stop.
> "make all" terminated with exit code 2. Build might be incomplete.

>

**Comment by anthony-kolesov** [comment]
Plugin was showing warning because it couldn't properly distinguish between situations of "importing a project for the older version of plugin" and "importing sources with makefile". Commit cb2d320693f2b1ecaca6b105820328a10603eef6 changes behavior, so now it will not show a warning, though now it will also not show it in some cases when it should be. However I've discovered that overall this mechanism doesn't work as it should, and may be should be removed. But at least for now it will not show incorrect warnings.

With respect of the "missing separator" issue, it doesn't seem like an IDE bug, because in this case makefile is not generated by IDE, and IDE has nothing to do with it - it just runs `make`. It seems to me that the problem could be with how embarc archive has been untarred (for example mixed windows and unix end lines), as a result "make" on Windows can't parse the file. Maybe it is something else, but it seems to be specific to sources and make used, because I have downloaded GNU 2020.03 RC1, mentioned embarc sources and I have no trouble building the project on my machine.
